gradle-plugin/src/main/java/org/projectnessie/nessierunner/gradle/NessieRunnerPlugin.java

```diff
@@ -13,14 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.projectnessie.quarkus.gradle;
+package org.projectnessie.nessierunner.gradle;
 
 import java.util.concurrent.ThreadLocalRandom;
 import org.gradle.api.Plugin;
 import org.gradle.api.Project;
 import org.gradle.api.provider.Provider;
 
-public class QuarkusAppPlugin implements Plugin<Project> {
+public class NessieRunnerPlugin implements Plugin<Project> {
 
   static final String EXTENSION_NAME = "nessieQuarkusApp";
```

Comment:
do these names matter / should be renamed?

we also seem to be using `nessie-quarkus-runner-` still as thread name below

Reply:
I'd like to keep these names, because those are used quite a lot - and technically not wrong - it's Nessie in Quarkus